public/category-sitemap.xml

```diff
@@ -1,299 +1,310 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
these were the files that were handed over to me. they passed on the w3c validator with a few warnings, so appear to be valid